phpunit.xml

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <phpunit bootstrap="../../tests/bootstrap.php"
-	 strict="true"
+	 beStrictAboutOutputDuringTests="true"
 	 verbose="true"
 	 failOnRisky="true"
 	 failOnWarning="true"
 >
-	<testsuite name='unit'>
-		<directory>./tests/unit</directory>
-	</testsuite>
+	<testsuites>
+		<testsuite name='unit'>
+			<directory suffix='Test.php'>./tests/unit</directory>

```

Comment:
Note: the unit test files are actually called `*Test.php` so I made this line like the other apps.

Reply:
Checked the unit test runs - they are running 32 unit tests before and after this change - good.